frontend/javascripts/admin/voxelytics/task_list_view.tsx

```diff
@@ -370,10 +371,8 @@ export default function TaskListView({
         message.error("Please select a specific run for log download.");
```

Comment:
It would be helpful to show this as a tooltip when the button is disabled, instead (or too).